docs/apache-airflow-providers-amazon/operators/rds.rst

```diff
@@ -166,6 +166,35 @@ To delete a AWS DB instance you can use
     :start-after: [START howto_operator_rds_delete_db_instance]
     :end-before: [END howto_operator_rds_delete_db_instance]
 
+.. _howto/operator:RdsStartDbOperator:
+
+Start a database instance or cluster
+====================================
+
+To start an AWS DB instance or cluster you can use
```

Comment:
```suggestion
To start an RDS DB instance or cluster you can use
```

Reply:
Actually to make it more consistent with the rest of the doc, I would go with

```suggestion
To start an Amazon RDS DB instance or cluster you can use
```